src/System.Text.Json/src/System/Text/Json/Node/JsonArray.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+
+#pragma warning disable CS1591
```

Comment:
I know I added this, but can we add a comment above to explain what warning are we disabling and why?

Reply:
OK, sure.